src/atomate2/common/jobs/phonons.py

```diff
@@ -230,7 +238,7 @@ def generate_frequencies_eigenvectors(
         Additional parameters that are passed to PhononBSDOSDoc.from_forces_born
 
     """
-    return PhononBSDOSDoc.from_forces_born(
+    phonon_doc = PhononBSDOSDoc.from_forces_born(
```

Comment:
you could directly return. I changed this recently.

Reply:
This was fixed by one of the linters. I was not sure if that was a style thing atomate2 did or not